tests/create_false/main.tf

```diff
@@ -0,0 +1,32 @@
+terraform {
+  required_version  = "~> 0.11.0"
+}
+
+provider aws {
+  region = "us-east-1"
+}
+
+resource "random_id" "name" {
+  byte_length = 6
+  prefix      = "terraform-aws-route53-querylog-"
+}
+
+module "route53_subzone" {
+  source = "../../"
+
+  providers = {
+    aws    = "aws"
+    aws.ns = "aws"
+  }
+
+  create_route53_subzone   		= false
```

Comment:
Got tabs in here, instead of spaces, in several places in this file...

Reply:
ahh yes, another benefit of testing this in Terraform itself is that I can fmt.